src/Main.ps1

```diff
@@ -899,6 +899,32 @@ function Invoke-Pester {
                 [PesterConfiguration] $PesterPreference = [PesterConfiguration]::Merge($callerPreference, $Configuration)
             }
 
+            if ($PesterPreference.Output.CIFormat.Value -eq 'Auto') {
```

Comment:
I didn't check `IsOriginalValue` here since its `Auto` by default. This will work for when its manually set or left by default.